docs/framework/configure-apps/file-schema/runtime/loadfromremotesources-element.md

```diff
@@ -25,75 +25,94 @@ Specifies whether assemblies from remote sources should be granted full trust.
+
+Starting with the .NET Framework 4, code access security (CAS) policy is disabled and assemblies are loaded in full trust. Ordinarily, this would grant full trust to assemblies loaded with the <xref:System.Reflection.Assembly.LoadFrom%2A?displayProperty=nameWithType> method that previously had been sandboxed. To prevent this, the ability to run code in assemblies loaded from a remote source is disabled by default. By default, if you attempt to load a remote assembly, a <xref:System.IO.FileLoadException> with an exception message like the following is thrown:
+
+```text
+System.IO.FileNotFoundException: Could not load file or assembly 'file:assem.dll' or one of its dependencies. Operation is not supported. 
+(Exception from HRESULT: 0x80131515)
+File name: 'file:assem.dll' ---> 
+System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly 
+to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, 
+so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. 
+```
+
+To load the assembly and execute its code, you must either:
+
+- Explicitly create a sandbox for the assembly (see [How to: Run Partially Trusted Code in a Sandbox](../../../../../docs/framework/misc/how-to-run-partially-trusted-code-in-a-sandbox.md)).
+
+- Run the assembly's code in full trust. You do this by configuring the `<loadFromRemoteSources>` element. It lets you specify that the assemblies that run in partial trust in earlier versions of the .NET Framework now run in full trust in the .NET Framework 4 and later versions.
+
 > [!IMPORTANT]
->  If the assembly should not run in full trust, do not set this configuration element. Instead, create a sandboxed <xref:System.AppDomain> in which to load the assembly.  
-  
-## Configuration File  
- This element is typically used in the application configuration file, but can be used in other configuration files depending upon the context. For more information, see the article [More Implicit Uses of CAS Policy: loadFromRemoteSources](http://go.microsoft.com/fwlink/p/?LinkId=266839) in the .NET Security blog.  
-  
-## Example  
- The following example shows how to grant full trust to applications from remote sources.  
-  
-```xml  
+> If the assembly should not run in full trust, do not set this configuration element. Instead, create a sandboxed <xref:System.AppDomain> in which to load the assembly.
+
+The `enabled` attribute for the `<loadFromRemoteSources>` element is effective only when code access security (CAS) is disabled. By default, CAS policy is disabled in the .NET Framework 4 and later versions. If you set `enabled` to `true`, remote assemblies are granted full trust.
+
+If `enabled` is not set to `true`, a <xref:System.IO.FileLoadException> is thrown under the following conditions:
```

Comment:
nit: should you specify if this means *either of the following conditions* or *both of the following conditions*?